lib/challenge.js

```diff
@@ -30,10 +56,14 @@
   // - a name
   // - an address
 
-var pet_owner;
 
-// Answer Starts Here
 
+// Answer Starts Here
+var pet_owner = {
+  ownerFirstName: "someone's firstName",
+  LastName: "someone's last name",
+  Address: "address"
+};
```

Comment:
Great. The only thing there is to note here is naming conventions. We don't name variables or properties with a capital initial unless they're constructor functions.

1